GitUI/UserControls/BlameControl.cs

```diff
@@ -279,49 +279,50 @@ private void ProcessBlame(string filename, GitRevision revision, IReadOnlyList<O
             // If it could be done with a solid rectangle around the text,
             // the extra spaces added here could be omitted.
 
+            var lineLength = Math.Max(80, 25 + _blame.Lines.Max(l => l.Commit.Author?.Length ?? 0) + _blame.Lines.Where(l => filename != l.Commit.FileName).Max(l => l.Commit.FileName.Length));
```

Comment:
Could you please wrap the line in a way that it becomes readable.
Right now it is near impossible to see what's going on.